### DISCUSSION TITLE
repopulating data in table

### DISCUSSION THREAD

**Discussion by 1fc0nfig** [grid-js/gridjs#863] [Q&A]
Making a copy of my [stackoverflow Q](https://stackoverflow.com/questions/70026070/repopulating-data-in-table-gridjs) here:
so i know the updateConfig() / forceRender() methods in gridjs, but is it possible to workaround the 'redrawing table'? I'm looking for a function/method that repopulate tables data with persisting view/search/filter options.

Thanks <3

**Comment by carlos-dubon** [comment]
I don't think the dev's thought of this. What I've done in my svelte project is to save and handle all those states (search, filters) by myself, and when a rerender occurs I have my own state to look at.
